src/app/payments/reimburse-fee.ts

```diff
@@ -1,6 +1,10 @@
-import { displayAmountFromNumber } from "@domain/fiat"
+import { DisplayAmountsConverter, displayAmountFromNumber } from "@domain/fiat"
 import { FeeReimbursement } from "@domain/ledger/fee-reimbursement"
-import { DisplayPriceRatio, WalletPriceRatio } from "@domain/payments"
+import {
+  DisplayPriceRatio,
```

Comment:
a bit confused as to why some parts of`Display` concept is in `payments` and some is in `fiat`. Not something to fix in this PR - just a remark

Reply:
I noticed that as well (was confusing at first too), but didn't want to touch it here. I'll probably try cleaning this up in a separate PR if it won't be a big change.